pkg/security/tests/activity_dumps_test.go

```diff
@@ -0,0 +1,184 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build functionaltests
+// +build functionaltests
+
+package tests
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/rules"
+)
+
+func TestActivityDumps(t *testing.T) {
+	ruleDefs := []*rules.RuleDefinition{
```

Comment:
Should not be necessary

Reply:
will be fixed by https://github.com/DataDog/datadog-agent/pull/12623,  I will rebase/remove this fake rule when the PR will be merged